src/mixins/nft.js

```diff
@@ -85,6 +85,19 @@ export default {
         this.iscnId
       )}%2F1`;
     },
+    populatedEvents() {
+      return this.NFTHistory.map(event => ({
+        ...event,
+        displayName: this.displayNameList[event.toWallet] || event.toWallet,
```

Comment:
`toDisplayName` is more suitable for this case
```suggestion
        toDisplayName: this.displayNameList[event.toWallet] || event.toWallet,
```